network-api/networkapi/buyersguide/templates/about/how_to_use.html

```diff
@@ -183,11 +183,19 @@ <h1 class="h2-heading">
     </p>
   </section>
 
+  <section aria-labelledby="guide-monetization">
+    <p id="guide-monetization"><b>Monetization of the Guide</b></p>
```

Comment:
it really feels like somewhere along the line we forgot to make sure we made these headings, rather than paragraphs with `<b>` tags - @kristinashu is there an appropriate heading class that we use if we turn these into `<h2>` or `<h3>` instead?

(then we can file a separate issue to do some replacing here, but `<b>`, `<i>`, `<s>`, etc shouldn't really be in HTML that also extensively uses CSS for its styling =D)

Reply:
Because there are so many titles on this page and each one only has a paragraph of text under it, we wanted to use the more subtle <b> instead of a heading style that would stand out too much.